nss-day-cohort-60/django-taters-api
# Lh categoryview

Closes #21


# Description of Proposed Changes
category_view added to rareapi/views
routers & path added to rare/urls
---


# Steps to Test

1. Fetch the `lh-categoryview branch and switch to it
2. Start/restart debugger
3. Open postman & add Authorization key with value Token 0be249c88238743e5b4a7ac370b5145730c28e20
4. Make a GET request http://localhost:8000/categories to verify that all 5 categories are listed
5. Make another GET request for each category using the PK to retrieve single category.
